tests/components/conversation/test_trigger.py

```diff
@@ -188,3 +190,56 @@ async def test_fails_on_punctuation(hass: HomeAssistant, command: str) -> None:
                 },
             ],
         )
+
+
+async def test_wildcards(hass: HomeAssistant, calls, setup_comp) -> None:
+    """Test wildcards in trigger sentences."""
+    assert await async_setup_component(
+        hass,
+        "automation",
+        {
+            "automation": {
+                "trigger": {
+                    "platform": "conversation",
+                    "command": [
+                        "play {album} by {artist}",
+                    ],
+                },
+                "action": {
+                    "service": "test.automation",
+                    "data_template": {"data": "{{ trigger }}"},
+                },
+            }
+        },
+    )
+
+    await hass.services.async_call(
+        "conversation",
+        "process",
+        {
+            "text": "play the white album by the beatles",
+        },
+        blocking=True,
+    )
+
+    await hass.async_block_till_done()
+    assert len(calls) == 1
+    assert calls[0].data["data"] == {
+        "alias": None,
+        "id": "0",
+        "idx": "0",
+        "platform": "conversation",
+        "sentence": "play the white album by the beatles",
+        "entities": {
+            "album": {
+                "name": "album",
+                "text": "the white album",
+                "value": "the white album",
+            },
```

Comment:
Entities?

Reply:
This is matching the sentence debug API, which uses hassil's terms. But both should arguably use "slots" instead to be in-line with the rest of HA.